### ISSUE TITLE
Movable Encumbrance HUD

### ISSUE THREAD

**Issue by O13SC3N3** [Insane-96/IguanaTweaksReborn#65]
Would like to be able to place this information where ever we see fit on the screen. Thank You, awesome to see Iguana Tweaks!

**Comment by Insane-96** [comment]
This is only possible on top left or top right

Il giorno Mar 15 Gen 2019, 06:12 O13SC3N3 <notifications@github.com> ha
scritto:

**Comment by O13SC3N3** [comment]
Oh, are we able to move it to the top left already? I haven't found the option to do so. Thank You

**Comment by Insane-96** [comment]
Nope, but as it's coded right now I could only move it to the left or to the right.
